.github/workflows/own-pr-review.yml

```
@@ -0,0 +1,53 @@
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PR="${{ github.event.pull_request.number }}"
+          REPO="${{ github.repository }}"
+          MARKER="<!-- rbk-own-pr-review-ci-stamp -->"
+          # Skip if already commented
+          if gh api "repos/$REPO/issues/$PR/comments" --jq '.[].body' | grep -q "$MARKER"; then
+            echo "stamp already present"
+            exit 0
+          fi
+          gh pr comment "$PR" --repo "$REPO" --body "$(cat <<MD
+$MARKER
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)